openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-dataextractor/src/tools/DataExtractor/DataExtractor.vue

```diff
@@ -830,7 +844,8 @@ export default {
             } else {
               let rawVal = packet[key]['raw']
               if (Array.isArray(rawVal)) {
-                row[columnMap[key]] = '"[' + rawVal + ']"'
+                // row[columnMap[key]] = '"[' + rawVal + ']"'
+                row[columnMap[key]] = 'BINARY'
```

Comment:
At a glance I thought this was outputting ARRAY items but they don't come across with 'raw' right? How would you extract raw binary?

Reply:
Array items aren't raw.   I'll write a ticket to get the JSON as an output.